Software/bsw/gen/CanIf_Cfg.h

```diff
@@ -245,86 +245,86 @@ typedef uint16 CanIfPublicHandleTypeEnum;
 //	Type: EcucStringParamDef
 //  Length: 1 - 32
 //*****************************************************************************
-#define CanIfInitCfgSet							(32U)
+#define CANIF_INIT_CFG_SET						(32U)
 
 /* Maximum total size of all Tx buffers. This parameter is needed only in
 case of post-build loadable implementation using static memory
 allocation.
 Range: 0..18446744073709551615 */
-#define CanIfMaxBufferSizeValue					(18446744073709551615)
+#define CANIF_MAX_BUFFER_SIZE					(18446744073709551615)
 
 /* Maximum number of Pdus. This parameter is needed only in case of
 post-build loadable implementation using static memory allocation.
 Range: 0..18446744073709551615 */
-#define CanIfMaxRxPduCfgValue					(18446744073709551615)
+#define CANIF_MAX_RX_PDU_CFG					(18446744073709551615)
 
 /* Maximum number of Pdus. This parameter is needed only in case of
 post-build loadable implementation using static memory allocation.
 Range: 0..18446744073709551615 */
-#define CanIfMaxTxPduCfgValue					(18446744073709551615)
+#define CANIF_MAX_TX_PDU_CFG					(18446744073709551615)
 
 /* CAN Identifier of transmit CAN L-PDUs used by the CAN Driver for
 CAN L-PDU transmission. Range: 11 Bit For Standard CAN Identifier
 ... 29 Bit For Extended CAN identifier
 The CAN Identifier may be omitted for dynamic transmit L-PDUs. */
-#define CanIfTxPduCanId0						(0U)
-#define CanIfTxPduCanId1						(1U)
+#define CANIF_TX_PDU_CAN_ID0					(0U)
```

Comment:
The same here, I believe it’s not 0 and 1

Reply:
i made 0 and 1 for multiplicity only and for the parameter usage is the same as i mentioned above in the Mask:
CanIfTxPduCanIdType parameter is the one that checks if its EXTENDED_CAN, EXTENDED_FD_CAN, STANDARD_CAN, STANDARD_FD_CAN.
not CanIfTxPduCanIdMask nor CanIfTxPduCanId.
what i understood from SWS is that they are used for MetaData. in SWS Page 35 (7.4 Dynamic L-PDUs).

and the below text is what in SWS in for the Dynamic L-PDUs:

CanIf shall support the ability to filter incoming messages using the CanIfRxPdu-
CanIdMask. The filtering shall be done by comparing the incoming CanId with the
stored CanIfRxPduCanId after applying the CanIfRxPduCanIdMask to both IDs.
This should be done after the filtering of regular CanIds without mask, to allow for
separate handling of some of the CanIds that fall into the range defined by the mask
or a CanId based range.
During transmission of dynamic L-SDUs, when a CanIfTxPduCanIdMask is defined,
the variable parts of the CanId provided via the MetaData must be merged with the
CanId by using this mask. When no CanIfTxPduCanIdMask and no CanIfTxPdu-
CanId are configured, the MetaData shall be used directly as CanId.